receiver/huaweicloudcesreceiver/internal/ces_client.go

```diff
@@ -0,0 +1,63 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package internal // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/huaweicloudcesreceiver/internal"
+
+import (
+	ces "github.com/huaweicloud/huaweicloud-sdk-go-v3/services/ces/v1"
+	"github.com/huaweicloud/huaweicloud-sdk-go-v3/services/ces/v1/model"
+)
+
+//go:generate mockery --name CesClient --case=underscore --output=../mocks
```

Comment:
will this be an issue when someone runs `go generate` on the project full project

we might need to minimally add some advice to install mockery tool on the `readme` file

potentially we may need to add a target to `makefile`

Reply:
not sure if the right place is the README. 
README is for the people who want to use the receiver. So I will rather add a comment on top of the interface.